js_modules/dagster-ui/tox.ini

```diff
@@ -20,18 +20,16 @@ allowlist_externals =
 commands =
   !windows: /bin/bash -c '! pip list --exclude-editable | grep -e dagster'
     yarn install
+  yarn workspace @dagster-io/ui-components ts
+  yarn workspace @dagster-io/ui-components lint:ci
```

Comment:
This is awesome!

A couple small ideas -- I think we can `lint:ci` without compiling, so we could potentially do that step first to catch basic errors even faster.  It also might be worth pulling Buildkite logs and identifying which packages fail most often and running those first. Running ui-components first because it adds the least time seems like a good improvement, but if ui-components isn't the source of many failed builds, it's fast but also low-value? Jumping straight to ui-core may make most of the failed builds in the last 30d fail faster.

Reply:
Fair points! Most people making changes in here are affecting `ui-core` and not `ui-components`, so it would be nice to surface those errors quickly.